creator-node/src/diskManager.js

```diff
@@ -2,7 +2,7 @@ const path = require('path')
 const fs = require('fs')
 const config = require('./config')
 const { logger: genericLogger } = require('./logging')
-const { CID } = require('ipfs-http-client-latest')
+const CID = require('cids')
```

Comment:
from @theoilie - code pointer for the underlying package in ipfs-http-client https://github.com/ipfs/js-ipfs/blob/524ff32901e43460fe5b364bc4903ab249792874/packages/ipfs-http-client/src/index.js#L3-L4